formik-mui/stories/components/DateTimePicker.js

```diff
@@ -1,8 +1,8 @@
 import React from 'react';
 import formikToMuiProps from '../../src/forms/formikToMuiProps';
-import {KeyboardDatePicker, KeyboardDateTimePicker} from '@material-ui/pickers';
-import InputAdornment from '@material-ui/core/InputAdornment';
-import MuiTextField from '@material-ui/core/TextField';
+import {DatePicker, DateTimePicker} from '@mui/lab';
```

Comment:
please check if KeyboardDatePicker's functionality are all included in DatePicker or not - esp. keyboard navigation

Reply:
https://github.com/settlin/node-monorepo/compare/master...aniket-teltumbade-au9:Muiv4tov5Migration?expand=1

Reply:
https://github.com/settlin/node-monorepo/compare/master...aniket-teltumbade-au9:Muiv4tov5Migration?expand=1